src/core/components/hv-navigator/index.tsx

```diff
@@ -334,9 +334,8 @@ export default class HvNavigator extends PureComponent<Props> {
 
     // Mutate the dom to match
     if (props.doc) {
-      const route: Element | null = props.doc.getElementById(
-        this.props.params.id,
-      );
+      const route: Element | null =
+        Dom.getElementById(props.doc, this.props.params.id) || null;
```

Comment:
do we need the `or` statement here?
```suggestion
        Dom.getElementById(props.doc, this.props.params.id);
```

Reply:
We do because getElementById returns "Element | null | undefined" which doesn't match "Element | null"